docs/libcudacxx/setup/building_and_testing.rst

```diff
@@ -27,176 +21,116 @@ In a Bash shell:
    apt-get -y install python-pip
    pip install lit
 
-   # Env vars that should be set, or kept in mind for use later
-   export LIBCUDACXX_ROOT=/path/to/libcudacxx # Git repo root.
-
-Step 1: Generate the Build Files
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-
-In a Bash shell:
-
-.. code:: bash
+Windows, Native Build/Test
+===========================
```

Comment:
Q: Are those instructions for native build/test on Windows (no linux), or for Windows and any native build/test on other OSs? Maybe to can rephrase the heading to make this more clear.